src/App.tsx

```diff
@@ -21,6 +22,14 @@ const App = () => {
             </AuthenticationGuard>
           }
         />
+        <Route
+          path="/"
+          element={
+            <AuthenticationGuard>
+              <Home />
+            </AuthenticationGuard>
+          }
+        />
```

Comment:
Ćwiczenie TypeScript:

Stwórz komponent (np `ProtectedRoute`) tak, aby przyjmował identyczne propsy jak Route (możesz najechać na Route i zobaczyć jak nazywa się interfejs przyjmowany przez Route + jeden dodatkowy prop `component`. Z tego `ProtectedRoute` zwróć `<Route>` z propsami oraz opakowany w <AuthenticationGuard>. Tutaj przyjmiesz po prostu zamiast użyć <Route> to uzyjesz <ProtectedRoute path="/" element={<Home />} /> a on juz opakuje w srodku potrzebnym guardem:)

Podejmujesz się?